firebase.json

```diff
@@ -206,7 +206,7 @@
       { "source": "/whats-new", "destination": "/release/whats-new", "type": 301 },
       { "source": "/testing/oem-debuggers", "destination": "/testing/native-debugging", "type": 301 },
 
-      { "source": "/adoptawidget", "destination": "https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md", "type": 301 },
+      { "source": "/adoptawidget", "destination": "https://github.com/flutter/flutter/blob/main/CONTRIBUTING.md", "type": 301 },
```

Comment:
As far as I can tell `master` is still the default channel on github.

Reply:
Yeah it is, but `main` also works and that's the intended end state, so I make sure to switch to it where possible. Helps us avoid making that transition any more challenging.

Let me know if you think it's important to change it back though!